README.adoc

```diff
@@ -1,3 +1,62 @@
-== Fabric8 Notification service
+= Fabric8 Notification service
+:toc:
+:toc-placement: preamble
 
+image:https://ci.centos.org/buildStatus/icon?job=devtools-fabric8-notification-build-master[Jenkins,link="https://ci.centos.org/view/Devtools/job/devtools-fabric8-notification-build-master/lastBuild/"]
+image:https://goreportcard.com/badge/github.com/fabric8-services/fabric8-notification[Go Report Card, link="https://goreportcard.com/report/github.com/fabric8-services/fabric8-notification"]
+image:https://godoc.org/github.com/fabric8-services/fabric8-notification?status.png[GoDoc,link="https://godoc.org/github.com/fabric8-services/fabric8-notification"]
+image:https://codecov.io/gh/fabric8-services/fabric8-notification/branch/master/graph/badge.svg[Codecov.io,link="https://codecov.io/gh/fabric8-services/fabric8-notification"]
 
+This service takes care of sending email notification for different Notification Type.
+
+=== Contributes:
+
+==== Project Structure:
+
+This project is build around Notification Type in center.
+
+To register new notification type and to see list of exisitng notification type check in [main.go](https://github.com/fabric8-services/fabric8-notification/blob/master/main.go)
+
+===== Introducing New Notification Type:
+
+====== Register:
+
+New Notification Type can be registered as below,
+
+```
+resolvers.Register("analytics.notify.cve", collector.ConfiguredVars(config, collector.NewCVEResolver(authClient, witClient)), nil)
+```
+
+First arg, Notification Type name (string), here it is "analytics.notify.cve".  You need to come up with unique Notification Type name depending on what kind of New Notificatoin you want to support.  Later value of this will be used to get registered Resolver and Validator in NotifyController.Send() 
```

Comment:
Can you link `Resolver` and `Validator` and `NotifyController.Send()` to the appropriate location wherever it is defined in the code?